### PR TITLE
r-annotationdbi: new package

### PR DESCRIPTION
**Pull request by lingnanyuan**
Create new packages r-annotationdbi and its dependencies: r-s4vectors, r-iranges.

### CODE REVIEW

**Comment by lingnanyuan** [comment]
The package also depends on r-rsqlite, I've add the pull request for this package.

**Comment by junghans** [comment]
Please, rebase on current develop branch!

File: var/spack/repos/builtin/packages/r-annotationdbi/package.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+#
+from spack import *
+
+
+class RAnnotationdbi(RPackage):
+    """Provides user interface and database connection code for
+ annotation data packages using SQLite data storage."""
```
Comment (junghans) [inline var/spack/repos/builtin/packages/r-annotationdbi/package.py]: Indentation

File: var/spack/repos/builtin/packages/r-iranges/package.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RIranges(RPackage):
+    """Provides efficient low-level and highly
+ reusable S4 classes for storing,
```
Comment (junghans) [inline var/spack/repos/builtin/packages/r-iranges/package.py]: Indentation.

File: var/spack/repos/builtin/packages/r-s4vectors/package.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RS4vectors(RPackage):
+    """The S4Vectors package defines the Vector and List virtual classes and
+ a set of generic functions that extend the semantic of ordinary vectors and
```
Comment (junghans) [inline var/spack/repos/builtin/packages/r-s4vectors/package.py]: Indentation.

**Pull request merged**